src/EnrollCallAndSmsController.js

```diff
@@ -136,7 +136,7 @@ function (Okta, FormController, Footer, PhoneTextBox, TextBox, CountryUtil, Form
               });
           };
 
-          if (isMfaEnroll) {
+          if (isMfaEnroll || !transaction.prev) {
```

Comment:
ah, i can remove this. I think I added this before I decided to merge `MFA_ENROLL` and `FACTOR_ENROLL` states and I was using this check to know it was new pipeline, since we dont send `prev` link in the new pipeline.